### PR TITLE
Add new colab notebooks inside examples folder

### PR DESCRIPTION
**Pull request by ggrace01**
Added new notebooks for: 
- stocks
- crypto 

Edited formatting syntax for: 
- options

### CODE REVIEW

**Review by sfhemstreet — APPROVED** [review]

File: examples/crypto-trading-basic.ipynb [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+   "metadata": {},
+   "source": [
+    "# Alpaca-py crypto trading basic"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/alpacahq/alpaca-py/blob/master/examples/cryoti-trading-basic.ipynb)"
```
Comment (hiohiohio) [inline examples/crypto-trading-basic.ipynb:14]:
```suggestion
    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/alpacahq/alpaca-py/blob/master/examples/crypto-trading-basic.ipynb)"
```

File: examples/crypto-trading-basic.ipynb [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# see all open orders\n",
+    "req = GetOrdersRequest(\n",
+    "    status = QueryOrderStatus.OPEN,\n",
+    "    symbols = [symbol]\n",
+    ")\n",
+    "open_orders = trade_client.get_orders(req)\n",
```
Comment (hiohiohio) [inline examples/crypto-trading-basic.ipynb:290]: Please consider separating cell to show this open_orders.

File: examples/stocks-trading-basic.ipynb [hunk 3]
```diff
@@ -0,0 +1,650 @@
...
+   "metadata": {},
+   "source": [
+    "# Alpaca-py stock trading basic"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/alpacahq/alpaca-py/blob/master/examples/options-trading-basic.ipynb)"
```
Comment (hiohiohio) [inline examples/stocks-trading-basic.ipynb:14]:
```suggestion
    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/alpacahq/alpaca-py/blob/master/examples/stocks-trading-basic.ipynb)"
```

File: examples/stocks-trading-basic.ipynb [hunk 4]
```diff
@@ -0,0 +1,650 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# see all open orders\n",
+    "req = GetOrdersRequest(\n",
+    "    status = QueryOrderStatus.OPEN,\n",
+    "    symbols = [symbol]\n",
+    ")\n",
+    "open_orders = trade_client.get_orders(req)\n",
```
Comment (hiohiohio) [inline examples/stocks-trading-basic.ipynb:418]: Please consider separating cell for this open_orders to show.

**Review by hiohiohio — COMMENTED** [review]
Overall LGTM! Left small comments.

**Review by hiohiohio — APPROVED** [review]
LGTM!

**Pull request merged**